test/runner/requirements/units.txt

```diff
@@ -4,6 +4,7 @@ cryptography
 pycrypto
 jinja2
 mock
+mocker
```

Comment:
I haven't been able to reproduce any requirements related unit test failures. What command are you running, and under what environment, that you're seeing errors relating to the `mocker` and `ncclient` requirements?